src/sentry/eventtypes/error.py

```diff
@@ -21,4 +22,7 @@ def get_metadata(self):
         }
 
     def to_string(self, metadata):
-        return u'{}: {}'.format(metadata['type'], metadata['value'])
+        return u'{}: {}'.format(
+            metadata['type'],
+            truncatechars(metadata['value'].splitlines()[0], 100),
```

Comment:
If I could recommend something more efficient:

```python
.split('\n', 1)[0]
```

because we only care about first line, so no need to split all of them.